player/lua/osc.lua

```diff
@@ -1755,7 +1786,7 @@ function osc_init()
             if (element.state.lastseek == nil) or
                 (not (element.state.lastseek == seekto)) then
                     mp.commandv("seek", seekto,
```

Comment:
The value of seekto is always percentage-based, so there's no use supporting non-percent modes.

Reply:
So that would mean "absolute", "-", and  "relative" are all useless correct? What about "keyframes" and "exact" as standalones and not paired with any other percent mode?

Reply:
Likely useless/broken too. In fact, only absolute-percent works with seekto as is. You'd need to change what value's passed to it for relative-percent to work.

Reply:
Single mouse-click on the seekbar already uses absolute-percent+exact, only dragging uses absolute-percent+keyframe for fast scrolling.
So, as is, I don't really see the point of this feature.

Reply:
Well, the original motivation came from someone pointing out that dragging the seekbar in videos with broken PTS generates some very horrifying artifacts. If you want to replicate it, you can download the big buck bunny movie and then put it in an ogm container with ogmmerge. With absolute-percent+exact, it seeks much better. Do you think would it be worth it to add only "absolute-percent+exact" as an option (and forget all the rest since they don't do anything) to cover that edge use case?

Reply:
Sure, if you mention it does make mouse dragging potentially a lot slower.

Reply:
Cool, I'll rebase it in a bit when I get a chance.